File: FirstApp/DevUse/SqlCommandTester.vb
Imports System.IO

Public Class sqlcommandtester
    Dim writer As StreamWriter
    Dim oprocess As New Process()
    Public value As String

    Private Sub Frmsimple_disposed(sender As Object, e As EventArgs) Handles Me.Disposed, Me.Closed
        Dim p = New Process()
        Dim pi = New ProcessStartInfo With {
                .Arguments = " /k" + " " + Command() + "taskkill /im python.exe /f",
                .FileName = "cmd.exe",
                .CreateNoWindow = True,
                .WindowStyle = ProcessWindowStyle.Hidden
                }
        p.StartInfo = pi
        p.Start()
    End Sub

    Private Sub Initialiseprocess()
        Dim s = ""
        Try
            s = File.ReadAllLines("c:\users\hp\desktop\tapustextfile.txt").ElementAt(0).ToString
        Catch ex As Exception
        End Try
        Try
            oprocess = Process.GetProcessById(Int(s))
            MessageBox.Show("using previous")
        Catch ' ex as argumentexception
            Dim ostartinfo As New ProcessStartInfo("c:\users\hp\desktop\pycharmprojects\tapuspycharm\helo.pyw") With {
                    .UseShellExecute = False,
                    .RedirectStandardInput = True,
                    .RedirectStandardOutput = True,
                    .RedirectStandardError = True,
                    .WindowStyle = ProcessWindowStyle.Hidden
                    }
            oprocess.StartInfo = ostartinfo
            oprocess.Start()
            File.WriteAllText("c:\users\hp\desktop\tapustextfile.txt", oprocess.Id.ToString)
        End Try
    End Sub

    Private Sub Form1_load(sender As Object, e As EventArgs) Handles MyBase.Load
        Initialiseprocess()
    End Sub

    Private Async Sub Button2_click(sender As Object, e As EventArgs) Handles Button2.Click
        oprocess.StandardInput.WriteLine("send")
        TextBox1.Text = Await oprocess.StandardOutput.ReadLineAsync
    End Sub
End Class

File: OrnaTrack/DevUse/SqlCommandTester.cs
using System;
using System.Diagnostics;
using System.IO;
using System.Linq;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace FirstApp
{

    public partial class sqlcommandtester
    {
        private StreamWriter writer;
        private Process oprocess = new Process();
        public string value;

        public sqlcommandtester()
        {
            InitializeComponent();
        }

        private void Frmsimple_disposed(object sender, EventArgs e)
        {
            var p = new Process();
            var pi = new ProcessStartInfo()
            {
                Arguments = " /k" + " " + Interaction.Command() + "taskkill /im python.exe /f",
                FileName = "cmd.exe",
                CreateNoWindow = true,
                WindowStyle = ProcessWindowStyle.Hidden
            };
            p.StartInfo = pi;
            p.Start();
        }

        private void Initialiseprocess()
        {
            string s = "";
            try
            {
                s = File.ReadAllLines(@"c:\users\hp\desktop\tapustextfile.txt").ElementAt(0).ToString();
            }
            catch (Exception ex)
            {
            }
            try
            {
                oprocess = Process.GetProcessById(Conversions.ToInteger(Conversion.Int(s)));
                MessageBox.Show("using previous"); // ex as argumentexception
            }
            catch
            {
                var ostartinfo = new ProcessStartInfo(@"c:\users\hp\desktop\pycharmprojects\tapuspycharm\helo.pyw")
                {
                    UseShellExecute = false,
                    RedirectStandardInput = true,
                    RedirectStandardOutput = true,
                    RedirectStandardError = true,
                    WindowStyle = ProcessWindowStyle.Hidden
                };
                oprocess.StartInfo = ostartinfo;
                oprocess.Start();
                File.WriteAllText(@"c:\users\hp\desktop\tapustextfile.txt", oprocess.Id.ToString());
            }
        }

        private void Form1_load(object sender, EventArgs e)
        {
            Initialiseprocess();
        }

        private async void Button2_click(object sender, EventArgs e)
        {
            oprocess.StandardInput.WriteLine("send");
            TextBox1.Text = await oprocess.StandardOutput.ReadLineAsync();
        }
    }
}